addons/binding/org.openhab.binding.enocean/ESH-INF/thing/CentralCommand.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="centralCommand">
```

Comment:
This does not sound as if it represents any kind of real device? Isn't this rather a channel of devices that also implement other EEPs at the same time?

Reply:
I have choosen the EEP family name from the enocean documentation for the thing-type ids. A central command device is a device which sends (central ;)) commands to different kinds of actuators like simple light switches, dimmers, fans or thermostats. As suggested I could also use the EEP number for the thing-type id.